src/main/kotlin/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+class RacingGame(
+    private var cars: Cars,
```

Comment:
```suggestion
    private val cars: Cars,
```
val로 작성해도 될 프로퍼티로 보입니다.